guides/hiring/remote_test/csharp_marking_guide.md

```diff
@@ -0,0 +1,28 @@
+# Marking guide
+
+60 marks in total. There are no half marks. 80% (48 marks) required to pass.
```

Comment:
> [2] Is the app built with Rails?
> [3] Rubocop is installed and is passing

Not all languages have tools of this nature although to be fair some languages absolutely need them - eslint for JS for example.

> [2] Scaffolding is not used for models
> [2] Scaffolding is not used for views
> [2] Scaffolding is not used for controllers

> [2] RSpec is used

You could specify a requirement for a BDD test framework but I'm not sure that really matters as long as the tests are readable and easy to understand. 

> [2] Capybara is used

Not sure about how absolutely necessary a gherkin syntax is for a project of this size or in general.

> [2] shoulda-matchers is used

I also added a 1 pointer about a nicely readable assertion library being used.

The thing I would be weary of is that most of these are more around getting the tooling set up, which is usually a one off task at the start of the project. I think the more important thing is making sure that the code is up to scratch quality wise and architected competently as that is the bulk of what they'll be doing day to day.